### ISSUE TITLE
7.3.2 to 7.4.0 upgrade: 404-Not Found, Value cannot be null

### ISSUE THREAD

**Issue by merijng** [abpframework/abp#17940]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Upgrade 7.3.2 partially fails.

### Reproduction Steps

- Create a new ABP project, version 7.3.2
- update the abp CLI to version 7.4.0 (`dotnet tool update -g Volo.Abp.Cli`)
- Navigate to the created ABP project directory using the Command Prompt (CMD)
- Update the project (`abp update -v 7.4.0`)
- Note that the upgrade for package Volo.Abp.AspNetCore.Mvc.UI. Theme.Shared failed

### Expected behavior

The package should update without any failures

### Actual behavior

An error, see screenshot above. Below text-based.

```
1. HTTP request attempt failed to https://nuget.abp.io/xxx/v3/package/Volo.Abp.AspNetCore.Mvc/index.json with an error: 404-Not Found. Waiting 2 secs for the next try...
```

```
Value cannot be null. (Parameter 'source')
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.All[TSource](IEnumerable`1 source, Func`2 predicate)
   at Volo.Abp.Cli.ProjectModification.VoloNugetPackagesVersionUpdater.SpecifiedVersionExists(String version, String packageId) in D:\ci\Jenkins\workspace\abp-volo-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\ProjectModification\VoloNugetPackagesVersionUpdater.cs:line 148
   at Volo.Abp.Cli.ProjectModification.VoloNugetPackagesVersionUpdater.UpdateVoloPackagesAsync(String content, Boolean includeNightlyPreviews, Boolean includeReleaseCandidates, Boolean switchToStable, SemanticVersion latestNugetVersion, SemanticVersion latestNugetReleaseCandidateVersion, String latestMyGetVersion, String specifiedVersion) in D:\ci\Jenkins\workspace\abp-volo-release\abp\framework\src\Volo.Abp.Cli.Core\Volo\Abp\Cli\ProjectModification\VoloNugetPackagesVersionUpdater.cs:line 297
```

### Version

7.3.2

### User Interface

MVC

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

### Workaround

Update `Volo.Abp.AspNetCore.Mvc` manually

![image](https://github.com/abpframework/abp/assets/6886678/704f9e21-ad6d-43ed-8bf2-2540671df504)

### Other information

I am currently in the process of executing the upgrade and will update this issue with additional details if necessary

**Comment by maliming** [comment]
hi

Please don't share your nuget key in public areas.

Are you using the commercial?

**Comment by ahmetfarukulu** [comment]
> Please don't share your nuget key in public areas.

~@maliming everyone can still see it in the edit history. We should delete this issue and create a new one.~

**Comment by merijng** [comment]
@ahmetfarukulu @maliming I've already deleted the original post. Excuse me, I thought it was just a public URL.

@maliming Yes, we use commercial.

**Comment by maliming** [comment]
hi @ahmetfarukulu @merijng 

Please create a question on [support.abp.io](https://support.abp.io/QA/Questions/New)

**Comment by merijng** [comment]
@maliming [here you go](https://support.abp.io/QA/Questions/6024/732-to-740-upgrade-404-Not-Found-Value-cannot-be-null)

Volo.Abp.AspNetCore.Mvc is not a commercial package.

**Comment by merijng** [comment]
This issue has been solved, see #17941.
